packages/app-headless-cms/src/admin/plugins/fieldRenderers/richText/utils.ts

```diff
@@ -0,0 +1,22 @@
+import { CmsModel, CmsModelField } from "@webiny/app-headless-cms-common/types";
+
+/**
+ * Model have at least one single/multiple legacy RTE field
+ */
+export const modelHasLegacyRteField = (model: CmsModel): boolean => {
+    return model.fields.some(field => field.renderer.name.startsWith("rich-text-"));
```

Comment:
Next function in your utils is this: `isLegacyRteField`. You can just pass that into `fields.some(isLegacyRteField)`, instead of duplicating the code.